obsplus/utils/mseed.py

```diff
@@ -1,19 +1,20 @@
 """
-Slimmed down functions for indexing miniseed files.
-
-Contains hacked bits from obspy's mseed module.
-
-Copyrights to obspy developers still apply.
+Useful description.
```

Comment:
TBD?

Reply:
haha, no this is part of my python file template I must have forgotten to fill in.